docs/SUMMARY.md

```diff
@@ -105,6 +105,8 @@
     * [Runtime dialect](explanations/RTDialect.md)
     * [SDFG dialect](explanations/SDFGDialect.md)
   * [Call FHE circuits from other languages](explanations/call_from_other_language.md)
+  * [Benchmarking](dev/benchmarking.md)
+  * [Examples](dev/examples.md)
```

Comment:
maybe "example" is not that clear, should we write "Tutorials" and "tutorials.md" everywhere? "example" would just be the name of the directory. 

Just a thought, we can let it as 'example' if you think it's clearer

Reply:
I think it's `Examples` is clearer because the directory is named `examples`. If we call this tutorials here, people wouldn't be able to find it in the repo directly.